src/main/kotlin/platform/mcp/aw/fixes/CopyAwAccessibleEntryFix.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2022 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.platform.mcp.aw.fixes
+
+import com.demonwav.mcdev.facet.MinecraftFacet
+import com.demonwav.mcdev.platform.fabric.FabricModuleType
+import com.demonwav.mcdev.platform.mcp.actions.CopyAwAction
+import com.demonwav.mcdev.util.findModule
+import com.intellij.codeInsight.daemon.QuickFixActionRegistrar
+import com.intellij.codeInsight.intention.IntentionAction
+import com.intellij.codeInsight.quickfix.UnresolvedReferenceQuickFixProvider
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.intellij.psi.PsiJavaCodeReferenceElement
+
+class CopyAwAccessibleEntryFix(val target: PsiElement, val element: PsiElement) : IntentionAction {
+
+    class Provider : UnresolvedReferenceQuickFixProvider<PsiJavaCodeReferenceElement>() {
```

Comment:
I wonder if this works with other JVM languages, or if we can use UAST here.

Reply:
I'll try it

Reply:
Reimplemented in UAST and tested in Kotlin, Scala, and Groovy; it works in none of them.
Quick fixes for invisible elements as unresolved elements only exist in Java; other languages all detect this differently and don't have ways of providing fixes, let alone a unified interface. I'm not sure if this can be provided for Kotlin at all without implementing visibility checks ourselves.

Reply:
Ah that's a shame, thanks for taking the time to try.